arches/app/templates/base-manager.htm

```diff
@@ -308,113 +316,140 @@ <h1 class="page-header text-overflow ep-graph-title">
 
                 </div>
 
-                <div class="col-xs-12 col-sm-4">
-
-                    <div role="navigation" class="top-right-nav">
+                <!-- Top Right Nav  -->
+                <nav class="col-xs-12 col-sm-4" id="user-actions-top-right-nav"> 
 
+                    <ul class="top-right-nav" aria-label="{% trans 'User and User Actions Navigation' %}">
+                        
                         <!-- Login -->
                         {% if nav.login %}
-                        <a href="{% if user.username != 'anonymous' %}{% url 'user_profile_manager' %}{% else %}{% url 'auth' %}?next={{ home }}{% endif %}"
-                            class="ep-tools ep-tools-login">
-                            <div data-placement="bottom"
-                                data-toggle="tooltip"
-                                data-original-title="{% if user.username != 'anonymous' %}{% trans 'Profile' %}{% else %}{% trans 'Login' %}{% endif %}">
-                                {% if user.username == 'anonymous' %}
-                                    <h5 class="hidden-xs">{% trans "Login" %}</h5>
-                                {% else %}
-                                    <h5 class="hidden-xs">{% trans "Welcome" %}, {{ user.first_name|default:user.username }}</h5>
-                                    <i class="fa fa-sign-out"></i>
-                                {% endif %}
-                            </div>
-                        </a>
+                        <li>
+                            <a href="{% if user.username != 'anonymous' %}{% url 'user_profile_manager' %}{% else %}{% url 'auth' %}?next={{ home }}{% endif %}"
+                                class="ep-tools ep-tools-login">
+                                <div data-placement="bottom"
+                                    data-toggle="tooltip"
+                                    data-original-title="{% if user.username != 'anonymous' %}{% trans 'Profile' %}{% else %}{% trans 'Login' %}{% endif %}">
+                                    {% if user.username == 'anonymous' %}
+                                        <span class="hidden-xs h5">{% trans "Login" %}</span>
+                                    {% else %}
+                                        <span class="hidden-xs h5" aria-label="{% trans 'Edit your profile' %}.">{% trans "Welcome" %}, {{ user.first_name|default:user.username }}</span>
```

Comment:
I think we might want to notify the user their logged in status as well as let then know that they can edit their profile.  Maybe it should say something like "Logged in as admin, click to edit your profile."